lib/plugin.zsh

```diff
@@ -4,7 +4,7 @@ typeset -g GEOMETRY_PLUGIN_SEPARATOR=${GEOMETRY_PLUGIN_SEPARATOR:-" "}
 # Define default plugins
 typeset -ga GEOMETRY_PROMPT_PLUGINS
 if [[ $#GEOMETRY_PROMPT_PLUGINS -eq 0 ]]; then
-  GEOMETRY_PROMPT_PLUGINS=(exec_time git)
+  GEOMETRY_PROMPT_PLUGINS=(exec_time git hg)
```

Comment:
Do we want to have this enabled be default? What if I don't have mercurial installed?

Wouldn't it be possible to at least check if it installed in the setup method?